Update Marketplace Release notes

JJdegroot1993 opened the pull request:
Release notes + small fix for Marketplace Release 12th of December
See also #8731 to be published at the same time.

Commit:
Update _index.md

Added a retention policy for free apps

Commit:
Merge branch 'mendix:development' into development

Commit:
Add files via upload

Luyao-Zhang-1 committed:
Edit

Commit:
Update _index.md

Retention policy of unused free apps

Commit:
Merge branch 'mendix:development' into development

Commit:
Update _index.md

Commit:
Update _index.md

Release notes Sprintr 2024.12.01

Commit:
Merge branch 'mendix:development' into development

Commit:
Update _index.md

Commit:
Update _index.md

Luyao-Zhang-1 committed:
review

Luyao-Zhang-1 committed:
review

jveldhuizen committed:
Update committing-objects.md with information about association reinserts for autocommitted objects

This information is added because of ticket 232998.

jveldhuizen committed:
Update committing-objects.md

katarzyna-koltun-mx committed:
Merge pull request #8730 from jveldhuizen/patch-6

Update committing-objects.md with information about association reinserts for autocommitted objects

MarkvanMents committed:
Add 10.17 Release Note about localized SP interface

MarkvanMents committed:
Proofread

MarkvanMents committed:
Merge pull request #8705 from loket4/development

Deletion of Unused Free Apps (to be released on December 1)

JJdegroot1993 committed:
Update general-marketplace.md

Release notes + small fix for Marketplace Release 5th of December

MarkvanMents committed:
Proofread

MarkvanMents commented on content/en/docs/releasenotes/marketplace/general-marketplace.md:
```diff
@@ -14,11 +14,19 @@ These release notes cover changes made to the [Mendix Marketplace](/appstore/).
+#### Improvements
+
+* Release notes are now editable for all versions of your Marketplace components. You no longer have to reach out to Mendix Support to change them.
+* The Company Subscriptions page has been removed from the Marketplace.
```
I assume that offboarded means the page has been removed (so not deprecated), or have I misunderstood?
I see various references to Company Subscriptions in the documents:
Should these be removed/updated?
https://docs.mendix.com/appstore/home-page/#company-subscriptions
https://docs.mendix.com/appstore/component-details/#header
https://docs.mendix.com/partners/siemens/3d-viewer/#check-usage

JJdegroot1993 replied:
Hi Mark,

Yes that is correct, the page has been removed. Thanks for bringing the other parts up. The other documentation parts can be removed as well.

JJdegroot1993 replied:
Hi @MarkvanMents I removed the information related to Company Subscriptions in the documents ;) 

Also, the release date will move to the 12th of December instead of the 5th, as the team has a small delay.

MarkvanMents replied:
Thanks for the update and letting me know about the changed date.

JJdegroot1993 committed:
Update marketplace-home-page.md

Removed (Company) Subscriptions from the Mendix Documentation as it will be removed from Marketplace

JJdegroot1993 committed:
Update component-details.md

Removed Company subscriptions as it will be removed from Marketplace

JJdegroot1993 committed:
Update _index.md

Removed section 5.9 as company subscriptions dashboard will be removed from Marketplace

MarkvanMents committed:
Update Marketplace release date

MarkvanMents reviewed (approved):
HI @JJdegroot1993,
Looks good. Approving.
Please let me know once the release is successful and I will merge.

MarkvanMents commented:
Changes should be combined with #8731 so merging them together.

MarkvanMents committed:
Merge branch 'lz-remove-company-subscription' into patch-5
